tests/handlebars/text-wrap/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -277,11 +277,7 @@ printWidth: 80
 
 {{! this really should split across lines }}
 <div>
-  before{{stuff}}after{{stuff}}after{{stuff}}after{{stuff}}after{{stuff}}after{{
-    stuff
-  }}after{{stuff}}after{{stuff}}after{{stuff}}after{{stuff}}after{{
-    stuff
-  }}after{{stuff}}after{{stuff}}after{{stuff}}after
+  before{{stuff}}after{{stuff}}after{{stuff}}after{{stuff}}after{{stuff}}after{{stuff}}after{{stuff}}after{{stuff}}after{{stuff}}after{{stuff}}after{{stuff}}after{{stuff}}after{{stuff}}after{{stuff}}after
```

Comment:
Is this intended?

Reply:
yes it is intended... was it prettier?

Reply:
I have no opinion on this. It just looked unrelated to the linked issue.

Reply:
I see, sorry for the confusion - I should have opened a specific issue for this PR